pkgs/development/python-modules/appdaemon/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, aiohttp, aiohttp-jinja2, astral, async, bcrypt, configparser, daemonize
+, feedparser, iso8601, jinja2, pyyaml, requests, sseclient
+, voluptuous, websocket-client, yarl }:
+
+buildPythonPackage rec {
+  pname = "appdaemon";
+  version = "3.0.0b3";
```

Comment:
Do you intentionally use a beta version?

Reply:
I started with the latest release version but there were dependency version conflicts.

Reply:
That will probably be a problem in the future as well. However, I think AppDaemon is never used as a library even though the `import` statements on http://appdaemon.readthedocs.io/en/stable/APPGUIDE.html suggest otherwise. These apps that import the `homeassistant.appdaemon` module are not executables on their own but imported by AppDaemon.

This means we can move `appdaemon` oy of `pythonPackages` without a problem. You might want to create an `appdaemon.nix` file in the `servers/home-assistant` directory.
Then, we can pin the versions of AppDaemon's dependencies to whatever is needed, just like in https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/default.nix.

I leave choosing the beta or the stable version up to you :)